lib/manageiq/environment.rb

```diff
@@ -93,6 +93,8 @@ def self.seed_database
     end
 
     def self.setup_test_environment(task_prefix: '', root: APP_ROOT)
```

Comment:
should the method name match the ENV var?
in other words, maybe method should be called `reset_test_environment`

Reply:
@chessbyte possibly?

I guess if you want to expand the scope of this PR, that is fine, I just would need to change `bin/update` and `bin/setup` as well.  A search seems to suggest it isn't called directly anywhere else besides this `ManageIQ/manageiq`:

https://github.com/search?q=org%3AManageIQ+setup_test_environment&type=Code

Reply:
I think based on this commit:

https://github.com/ManageIQ/manageiq/commit/045784d

This was named because previously the `puts` output was "Preparing tests".

Anyway, @bdunne , since this is your handywork, I assume you have no objections to the name change?

Reply:
I'm 👍  on coming up with better names for the ENV variables or methods but considering people might have existing env vars in their shells and SKIP_AUTOMATE_RESET probably has the same concerns, it might make sense to revisit that as a followup PR since naming could take several revisions to get good names 🤣 .